client/asset/eth/eth.go

```diff
@@ -328,6 +329,21 @@ func (w *TokenWallet) Info() *asset.WalletInfo {
 	}
 }
 
+// genWalletSeed uses the wallet seed passed from core as the entropy for
```

Comment:
maybe elaborate a bit: `uses the wallet seed passed from core` -> `uses the wallet seed core derives from app seed`